README.md

```diff
@@ -1,29 +1,33 @@
-# pm <a href='https://degauss.org'><img src='https://github.com/degauss-org/degauss_template/raw/master/DeGAUSS_hex.png' align='right' height='138.5' /></a>
+# pm <a href='https://degauss.org'><img src='https://github.com/degauss-org/degauss_hex_logo/raw/main/PNG/degauss_hex.png' align='right' height='138.5' /></a>
 
-> add daily PM2.5 estimates to geocoded data
+[![](https://img.shields.io/github/v/release/degauss-org/pm?color=469FC2&label=version&sort=semver)](https://github.com/degauss-org/pm/releases)
+[![container build status](https://github.com/degauss-org/pm/workflows/build-deploy-release/badge.svg)](https://github.com/degauss-org/pm/actions/workflows/build-deploy-release.yaml)
 
-[![GitHub Latest Tag](https://img.shields.io/github/v/tag/degauss-org/pm)](https://github.com/degauss-org/pm/releases)
+## Using
 
-## DeGAUSS example call
-
-If `my_address_file_geocoded.csv` is a file in the current working directory with coordinate columns named `lat` and `lon` and date columns called `start_date` and `end_date` then
+If `my_address_file_geocoded.csv` is a file in the current working directory with coordinate columns named `lat`, `lon`,  `start_date`, and `end_date` then the [DeGAUSS command](https://degauss.org/using_degauss.html#DeGAUSS_Commands):
 
 ```sh
-docker run --rm -v $PWD:/tmp ghcr.io/degauss-org/pm:0.1.3 my_address_file_geocoded.csv
+docker run --rm -v $PWD:/tmp ghcr.io/degauss-org/pm:0.2.0 my_address_file_geocoded.csv
 ```
 
-will produce `my_address_file_geocoded_pm_v0.1.2.csv` with added columns named `pm_pred` and `pm_se`, and a row for each date between the provided `start_date`s and `end_date`s. 
+will produce `my_address_file_geocoded_pm_0.2.0.csv` with added columns:
```

Comment:
Let's leave off "with added columns" here since that makes it seem like the rows will be unchanged (which they usually are, but not in the special case with dates).

Reply:
So just

```
will produce `my_address_file_geocoded_pm_0.2.0.csv`. 

- **`pm_pred`**: predicted PM2.5 (micrograms per cubic meter)
- **`pm_se`**: standard error for predicted PM2.5
```

Or should we add something else to explain these are the column names and definitions? 

Or we could move the part about 1 row per day up before the column names.

```
will produce `my_address_file_geocoded_pm_0.2.0.csv`. 

The output file will contain one row per day between `start_date` and `end_date` for each individual `lat` and `lon` location. This means that the output file will likely contain many more rows than the input file, so using identifiers with this container is useful for merging its output with other sources. 

- **`pm_pred`**: predicted PM2.5 (micrograms per cubic meter)
- **`pm_se`**: standard error for predicted PM2.5
```

Reply:
Yes, I like the idea of moving up the explanation before the column names instead.